CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 ## Current Develop Branch
 
 ## 8.0.6 Wed Jul 22 2020
+- [#9030](https://github.com/MetaMask/metamask-extension/pull/9030): Hide "delete" button when editing contact of wallet account
+- [#9031](https://github.com/MetaMask/metamask-extension/pull/9031): Fix crash upon removing contact
+- [#9032](https://github.com/MetaMask/metamask-extension/pull/9032): Do not show spend limit for approvals
+- [#9046](https://github.com/MetaMask/metamask-extension/pull/9046): Update @metamask/inpage-provider@6.1.0
+- [#9048](https://github.com/MetaMask/metamask-extension/pull/9048): Skip attempts to resolve 0x contract prefix
+- [#9051](https://github.com/MetaMask/metamask-extension/pull/9051): Use content-hash@2.5.2
+- [#9056](https://github.com/MetaMask/metamask-extension/pull/9056): Display three significant digits for token balances
```

Comment:
Oh, upon reviewing this change again, this is an inaccurate way to describe this. I'll try again.

Reply:
Maybe this? cc @brad-decker 
```suggestion
- [#9056](https://github.com/MetaMask/metamask-extension/pull/9056): Display at least one significant digit of small non-zero token balances
```

Reply:
I'll go with this for now anyway

Reply:
I think the suggestion you wrote works well @Gudahtt